webfront/serializers/content_serializers.py

```diff
@@ -298,3 +302,18 @@ def add_pagination(self, payload, obj):
         if "before_key" in obj and obj["before_key"] is not None:
             previous = replace_query_param(url, "cursor", obj["before_key"])
         return {"next": next_page, "previous": previous, **payload}
+
+    @staticmethod
+    def generic_get_counters(main_endpoint, counter_endpoints, instance, searcher, queryset_manager):
+        sq = queryset_manager.get_searcher_query()
+        counters = {}
+        for ep in counter_endpoints:
+            if (
+                ep not in queryset_manager.filters or(
+                "accession" not in queryset_manager.filters[ep]
+                and "accession__iexact" not in queryset_manager.filters[ep])
+            ):
+                counters[counter_endpoints[ep][0]] = searcher.get_number_of_field_by_endpoint(
+                    main_endpoint, counter_endpoints[ep][1], instance.accession, sq
```

Comment:
Use the `.items()` method to be able to direct access values:

```suggestion
        for ep, (name, field) in counter_endpoints.items():
            if (
                ep not in queryset_manager.filters or(
                "accession" not in queryset_manager.filters[ep]
                and "accession__iexact" not in queryset_manager.filters[ep])
            ):
                counters[name] = searcher.get_number_of_field_by_endpoint(
                    main_endpoint, field, instance.accession, sq
```